### ISSUE TITLE
readme example no longer working as written.

### ISSUE THREAD

**Issue by hanetzer** [PSPReverse/amd-nv-tool#1]
Guessing some code changes happened between readme creation and today.

```bash
$ amdnvtool example.nvdata
Error: more arguments are required
usage: amdnvtool romfile [-s <secret_hex> | lsb_key_hex]
```

**Comment by cwerling** [comment]
You're right, sorry. This was from an intermediate version of the tool where we didn't decrypt the NVDATA yet. In its latest form, the tool expects one of two forms of key material to decrypt the data. You can see the end-to-end attack flow in the Readme over here: https://github.com/PSPReverse/ftpm_attack

**Comment by hanetzer** [comment]
Ah, I was under the impression these had to do with 'bios settings', but its good to know I'm not doing something wrong here. Not really in my area of interest, but cool.
